packages/gatsby-source-wordpress/src/steps/create-schema-customization/helpers.js

```diff
@@ -84,7 +116,7 @@ export const typeIsASupportedScalar = type => {
   return supportedScalars.includes(findTypeName(type))
 }
 
-const typeSettingCache = {}
+const typeSettingCache = new Map()
```

Comment:
It's better to use a Map for caching since this function is called tons of times during schema customization and query building